verification/client.go

```diff
@@ -76,18 +128,51 @@ func (c *Client) GetProfile() (*GetProfileResp, error) {
 	}, nil
 }
 
+func (c *Client[_, _]) GetProfile() (*GetProfileResp, error) {
+	return getProfile(c.transport)
+}
+
 // Send the command to destroy a context.
-func (c *Client) DestroyContext(cmd *DestroyCtxCmd) error {
+func (c *Client[_, _]) DestroyContext(cmd *DestroyCtxCmd) error {
 	// DestroyContext does not return any parameters.
 	respStruct := struct{}{}
 
-	if _, err := execCommand(c.transport, CommandInitializeContext, c.Profile, cmd, &respStruct); err != nil {
+	if _, err := execCommand(c.transport, CommandDestroyContext, c.Profile, cmd, &respStruct); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+// CertifyKey calls the DPE CertifyKey command.
+func (c *Client[CurveParameter, Digest]) CertifyKey(cmd *CertifyKeyReq[Digest]) (*CertifyKeyResp[CurveParameter, Digest], error) {
+	// Define an anonymous struct for the response, because we have to accept the variable-sized certificate.
+	respStruct := struct {
+		NewContextHandle  [16]byte
+		DerivedPublicKeyX CurveParameter
+		DerivedPublicKeyY CurveParameter
+		CertificateSize   uint32
+		Certificate       [2048]byte
+	}{}
```

Comment:
I'm still pretty new to Go, is possible to do something like this to avoid having two very similar structs?

```suggestion
	respStruct := CertifyKeyResp[CurveParameter, Digest]{}
```